### DISCUSSION TITLE
macOS and mobile phone can access the internet,but Windows PC can't.

### DISCUSSION THREAD

**Discussion by wxp11** [Mygod/VPNHotspot#437] [Q&A]
Using this hotspot,I found that macOS and mobile phone can access the internet,but Windows PC can't.

**Comment by nibazshab** [comment]
At my devices, Windows and iPhone access the internet as usual

**Comment by SoulGI** [comment]
I have the same problem as well, although I have tried the newest version 2.16.10. My other devices works properly instead of WIndows PC, keeping showing me "the connection was reset"--a sign of having issues to connect to VPN. I have tried to disable Tethering hardware acceleration, restart my VPN, but nothing happened.
